src/it/projects/mdep-586-unpacking-resources/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.apache.maven.plugins.dependency.its</groupId>
+    <artifactId>mdep-586-unpacking-resource</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <description>Unpacking different files in the same JAR in different locations</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>test</groupId>
+            <artifactId>ResourceArtifact</artifactId>
+            <version>1.0</version>
+        </dependency>
+    </dependencies>
+
+    <repositories>
```

Comment:
Can you replace the jar by creating the directory `src/it/mrm/repository/ResourceArtifact-1.0.jar`, put its resources in that directory and add a `ResourceArtifact-1.0.pom`. This way we don't need to depend on binaries and the processflow is more like the real world.
All this is possible because of https://www.mojohaus.org/mrm/mrm-maven-plugin/examples/invoker-tests.html

Reply:
After that this repository-block can be removed.

Reply:
I'm trying to do that, but the generated JAR is empty. For now I am checking how mrm-maven-plugin works.

Reply:
I'm checking what I'm missing first.

Reply:
My guess: update to latest version of mrm-maven-plugin

Reply:
JAR still being generated without resource1.txt and resource2.txt. I also attempted to create the folder src/main/resources and replicate these files there.